### PR TITLE
implemented new design for specific job page

### PR DESCRIPTION
**Pull request by linda9379**
image

updated buttons since we dont have a shortlist feature right now

### CODE REVIEW

File: client/src/views/SpecificJobPage/SpecificJobPage.tsx [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+          </MainWrapper>
+        </Container>
+      </WaterWrapper>
+    </>
+  )
+}
+
+const MainWrapper = styled.div`
+  display: flex;
+  align-items: left;
```
Comment (a28dong) [inline client/src/views/SpecificJobPage/SpecificJobPage.tsx:298]: random q:
what does align-items left do?

was tryna search it up but idt it exists?

Comment (linda9379) [reply client/src/views/SpecificJobPage/SpecificJobPage.tsx:298]: it aligns all the components in the wrapper to the left

File: client/src/views/SpecificJobPage/SpecificJobPage.tsx [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+          )
+        })}
+    </Box>
+  )
+  const StyledTab = styled(Tab)({
+    '&.MuiTab-textColorPrimary': {
+      color: 'white',
+    },
+  })
+  const StyledShortlistButton = styled(MUIButton)({
```
Comment (a28dong) [inline client/src/views/SpecificJobPage/SpecificJobPage.tsx:203]: are other buttons in our design looking the same?
In general, I think we should probably start creating reuseable components when applicable

File: client/src/views/SpecificJobPage/SpecificJobPage.tsx (hunk 1 above)
Comment (a28dong) [reply client/src/views/SpecificJobPage/SpecificJobPage.tsx:298]: shouldn't it be `flex-start`?
idk im a css noob but I never heard of `left`

**Comment by a28dong** [comment]
lgtm

File: client/src/views/SpecificJobPage/SpecificJobPage.tsx (hunk 1 above)
Comment (linda9379) [reply client/src/views/SpecificJobPage/SpecificJobPage.tsx:298]: lmaoo mb i'll change that i didnt really goggle it

**Review by a28dong — APPROVED** [review]
lgtm

**Pull request merged**